po/es_AR.po

```diff
@@ -4,18 +4,16 @@
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
 #
 
-#, fuzzy
-
```

Comment:
Apparently we should be regnerating the messages file in the first place, but I think it's harmless:

 * https://www.gnu.org/software/gettext/manual/html_node/Fuzzy-Entries.html#Fuzzy-Entries